build/rule-generator/get-files-metadata.js

```diff
@@ -60,8 +60,13 @@ const getRuleMatchesFileMeta = (
 		const ruleMatchesJs = {
 			name: `${ruleName}-matches.js`,
 			content: `
-			// TODO: Filter node(s)
-			return node;
+			// TODO: Filter node(s)	
+			
+			function getMatches(node) {
```

Comment:
Would be great if this function was named base don the rule for you but using camel case. So a rule named `my-rule` would produce `myRuleMatches`